onnxscript/function_libs/torch_lib/ops/nn.py

```diff
@@ -2197,16 +2197,86 @@ def aten_unflatten_dense_tensors(
     raise NotImplementedError()
 
 
+@torch_op("aten::upsample_bicubic2d", trace_only=True)
 def aten_upsample_bicubic2d(
-    self: TensorType,
+    self: TReal,
     output_size: INT64,
     align_corners: bool,
-    scales_h: Optional[float] = None,
-    scales_w: Optional[float] = None,
-) -> TensorType:
-    """upsample_bicubic2d(Tensor self, SymInt[2] output_size, bool align_corners, float? scales_h=None, float? scales_w=None) -> Tensor"""
+    scales: TFloat = None,
+) -> TReal:
+    """upsample_bicubic2d.vec(Tensor input, SymInt[]? output_size, bool align_corners, float[]? scale_factors) -> Tensor"""
+    """upsample_bicubic2d(Tensor self, SymInt[2] output_size, bool align_corners, float? scales_h=None, float? scales_w=None) -> Tensor"""  # pylint: disable=pointless-string-statement
```

Comment:
Should we split them into two functions?

Reply:
Onnx.Resize() only accept either output_size or scales, it is error if both provided.
So for the second signature, when the ```output_size``` have been provided, the scales_h and scales_w will be ignored

Reply:
Do we handle when `scales_h` and `scales_w` are not None for `upsample_bicubic2d`?

Reply:
Didn't handled.
Because from this signature ```upsample_bicubic2d(Tensor self, SymInt[2] output_size, bool align_corners, float? scales_h=None, float? scales_w=None) -> Tensor```, the ```output_size``` must be provided, then we will ignore ```scales_h``` and ```scales_w```.